src/tool/subcommands/api_cmd/test_snapshots.txt

```diff
@@ -143,6 +143,14 @@ filecoin_statedecodeparams_1753985928338431.rpcsnap.json.zst
 filecoin_statedecodeparams_1753985928337192.rpcsnap.json.zst
 filecoin_statedecodeparams_1753985928337443.rpcsnap.json.zst
 filecoin_statedecodeparams_1753985928337618.rpcsnap.json.zst
+filecoin_statedecodeparams_1754479441718517.rpcsnap.json.zst
```

Comment:
We need to rename these files by adding the `actor name` and re-upload them.

for ex: `filecoin_verifie_reg_statedecodeparams_1754230255606413.rpcsnap.json.zst`

Reply:
fixed